src/protocolgame.cpp

```diff
@@ -3236,12 +3236,12 @@ void ProtocolGame::sendPodiumWindow(const Item* item)
 
 	// add GM outfit for staff members
 	std::vector<ProtocolOutfit> protocolOutfits;
+	protocolOutfits.reserve(outfits.size());
```

Comment:
Reserve can be removed, it's not that many outfits and not too often called to warrant